src/useSynthetix.js

```diff
@@ -13,7 +13,8 @@ function updateObject(oldObject, newValues) {
 const SYNTHETIX_INITIAL_STATE = Object.freeze({
   walletAddress: undefined,
   connect: () => {},
-  signature: undefined,
+  tokens: {},
```

Comment:
```suggestion
  token: undefined,
```
No need for the app to know any potential tokens for other wallet addresses

Reply:
keep only one token in app